src/registry/metadataRegistry.json

```diff
@@ -4639,6 +4644,49 @@
       "directoryName": "extlClntAppPushPolicies",
       "inFolder": false,
       "strictDirectoryName": false
+    },
+    "appframeworktemplatebundle": {
```

Comment:
I only added this guy. Scripts added the rest.

Reply:
might want to check with the cli team on these additions.  Is it normal for scripts to add unrelated stuff. Leave or remove?